### DISCUSSION TITLE
It's not working in my GTR4 watch for fetching internet data

### DISCUSSION THREAD

**Discussion by bojinhong** [zepp-health/zeppos-forum#35] [Q&A] [answered]
Hi,

I modify the example code(FetchAPI) to fetch real internet data. In. the simulator, everything work fine. But when using developer bridge mode to install app to the GTR4, it's not working to fetch the data. Please give me some direction to debug this issue, thanks.

**Comment by sirAvent** [comment]
Hi! Is there any message in the console when you fetch on your GTR4?

**Reply by bojinhong** [reply]
Hi, I just found out it is the api response Json format issue. I think I have no problem now. Thanks for quick response. It is my first to develop zepp os app.

**Comment by wimva** [comment, marked as answer]
Did you already try on both iOS and Android? There are some differences in catching the body responses on Android vs simulator and iOS. Make sure to do plenty of logs and see where it fails. I debugged this with showing log info on the watch display itself temporary.

**Reply by bojinhong** [reply]
Hi, my problem is fixed. Thanks a lot.
